apis/v1/gateway_types.go

```diff
@@ -390,6 +390,9 @@ type Listener struct {
 	// same port, subject to the Listener compatibility rules.
 	//
 	// Support: Core
+	//
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Maximum=65535
```

Comment:
these annotations don't seem to work here for some reason which is confusing

Reply:
@dprotaso it looks like this works and the only unexpected difference in verify result is in backend policies and routes which I think just means that this needs to be copied more places:

https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_gateway-api/3750/pull-gateway-api-verify/1911912967000035328